src/block-components/columns/column-settings-button.js

```diff
@@ -16,6 +18,8 @@ export const ColumnsControl = ( { sliderMax = 6 } ) => {
 		numInnerBlocks, innerBlocks,
 	} = useBlockContext()
 	const [ isDuplicate, setIsDuplicate ] = useLocalStorage( 'stk__columns_new_duplicate', false )
+	const [ localNumInnerBlocks, setLocalNumInnerBlocks ] = useLocalStorage( 'stk__columns_num_inner_blocks' )
+	const [ localInnerBlocks, setLocalInnerBlocks ] = useLocalStorage( 'stk__columns_inner_blocks' )
```

Comment:
You don't need to use `useLocalStorage` here, this can be just `useState` because you don't need this to persist